tests/system/marathon_pods_tests.py

```diff
@@ -505,18 +506,52 @@ def test_pod_with_persistent_volume():
     host = common.running_status_network_info(tasks[0]['statuses'])['ip_addresses'][0]['ip_address']
     port1 = tasks[0]['discovery']['ports']['ports'][0]["number"]
     port2 = tasks[1]['discovery']['ports']['ports'][0]["number"]
-    dir1 = tasks[0]['container']['volumes'][0]['container_path']
-    dir2 = tasks[1]['container']['volumes'][0]['container_path']
-    print(host, port1, port2, dir1, dir2)
+    path1 = tasks[0]['container']['volumes'][0]['container_path']
+    path2 = tasks[1]['container']['volumes'][0]['container_path']
+    print(host, port1, port2, path1, path2)
 
-    time.sleep(1)
+    @retrying.retry(wait_fixed=1000, stop_max_attempt_number=30, retry_on_exception=common.ignore_exception)
+    def check_http_endpoint(port, path):
```

Comment:
This is very similar to `check_task`. Should we unify then?

Reply:
Too soon.